### PR TITLE
Fix a typo

### PR DESCRIPTION
**Pull request by nabeel-InfoTrack**
## Summary

This Pull Request will fix a small typing error in the original article. "_that the topic could **easy** take several books_" should be written like "_that the topic could **easily** take several books_".

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/architecture/microservices/secure-net-microservices-web-applications/index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ ms.date: 08/07/2020
 ---
 # Make secure .NET Microservices and Web Applications
 
-There are so many aspects about security in microservices and web applications that the topic could easy take several books like this one so, in this section, you'll focus on authentication, authorization, and application secrets.
+There are so many aspects about security in microservices and web applications that the topic could easily take several books like this one so, in this section, you'll focus on authentication, authorization, and application secrets.
```
Comment (mairaw) [inline docs/architecture/microservices/secure-net-microservices-web-applications/index.md:9]:
```suggestion
There are so many aspects about security in microservices and web applications that the topic could easily take several books like this one. So, in this section, we'll focus on authentication, authorization, and application secrets.
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @nabeel-InfoTrack. I've noticed this sentence could be further improved. Do you mind applying my suggestion as well? Thanks

**Comment by gewarren** [comment]
@mairaw Can you recheck this one?

**Review by mairaw — APPROVED** [review]
Thanks @nabeel-InfoTrack!

**Pull request merged**